release/createAdoPrs.js

```diff
@@ -168,7 +168,7 @@ async function main()
         util.verifyMinimumNodeVersion();
         util.verifyMinimumGitVersion();
         createIntegrationFiles(newRelease);
-        util.execInForeground(`${GIT} config --global user.email "${process.env.USER}@microsoft.com"`, null, opt.dryrun);
+        util.execInForeground(`${GIT} config --global user.email "${process.env.EMAIL}"`, null, opt.dryrun);
```

Comment:
Nit: useremail?

Reply:
thanks, changed USER to USERNAME and EMAIL to USEREMAIL